src/logger/logger.go

```diff
@@ -14,5 +17,49 @@ func InitLogger() *logrus.Logger {
 		HideKeys:    true,
 		FieldsOrder: []string{"category"},
 	})
+
 	return log
 }
+
+// MiddleWare ...
+// logs the requests
+func MiddleWare(log *logrus.Logger) gin.HandlerFunc {
```

Comment:
영어 공부좀... 그리고 함수위에 주석 쓰려면 https://blog.golang.org/godoc godoc스타일대로 쓰는게 좋아보임
```suggestion
func Middleware(log *logrus.Logger) gin.HandlerFunc {
```